src/pybind/mgr/restful/common.py

```diff
@@ -6,7 +6,7 @@
 
 # Implemented osd commands
 OSD_IMPLEMENTED_COMMANDS = [
-    'scrub', 'deep-scrub', 'repair'
+    'scrub', 'deep-scrub', 'deep_scrub', 'repair'
```

Comment:
Was ever `deep_scrub` exposed here?

Reply:
yes - but (starting at some point in the past, when someone "fixed" this file) not with the common command format (tell [pg] deep-scrub).

Reply:
But I agree with you and Neha's comment below: as this was not supported in the last two versions,
I will run the upgrade tests without it.